executors.go

```diff
@@ -42,6 +59,25 @@ func (q *Query) ExecWithCount() (int, error) {
 // ValidateAndSave applies validation rules on the given entry, then save it
 // if the validation succeed, excluding the given columns.
 func (c *Connection) ValidateAndSave(model interface{}, excludeColumns ...string) (*validate.Errors, error) {
+	v := reflect.Indirect(reflect.ValueOf(model))
```

Comment:
I feel like a lot of this logic is repeated. would make sense to break it out into it's own method?